docs/bpftool-btf.rst

```diff
@@ -35,161 +35,161 @@ BTF COMMANDS
-		  associated BTF object with BTF types.
+		When **prog** is provided, it's expected that program has
+		associated BTF object with BTF types.
 
-		  When specifying *FILE*, an ELF file is expected, containing
-		  .BTF section with well-defined BTF binary format data,
-		  typically produced by clang or pahole.
+		When specifying *FILE*, an ELF file is expected, containing
+		.BTF section with well-defined BTF binary format data,
+		typically produced by clang or pahole.
 
-		  **format** option can be used to override default (raw)
-		  output format. Raw (**raw**) or C-syntax (**c**) output
-		  formats are supported.
+		**format** option can be used to override default (raw)
+		output format. Raw (**raw**) or C-syntax (**c**) output
+		formats are supported.
 
 	**bpftool btf help**
-		  Print short help message.
+		Print short help message.
 
 OPTIONS
 =======
 	.. include:: common_options.rst
 
 	-B, --base-btf *FILE*
-		  Pass a base BTF object. Base BTF objects are typically used
-		  with BTF objects for kernel modules. To avoid duplicating
-		  all kernel symbols required by modules, BTF objects for
-		  modules are "split", they are built incrementally on top of
-		  the kernel (vmlinux) BTF object. So the base BTF reference
-		  should usually point to the kernel BTF.
-
-		  When the main BTF object to process (for example, the
-		  module BTF to dump) is passed as a *FILE*, bpftool attempts
-		  to autodetect the path for the base object, and passing
-		  this option is optional. When the main BTF object is passed
-		  through other handles, this option becomes necessary.
+		Pass a base BTF object. Base BTF objects are typically used
+		with BTF objects for kernel modules. To avoid duplicating
+		all kernel symbols required by modules, BTF objects for
+		modules are "split", they are built incrementally on top of
+		the kernel (vmlinux) BTF object. So the base BTF reference
+		should usually point to the kernel BTF.
+
+		When the main BTF object to process (for example, the
+		module BTF to dump) is passed as a *FILE*, bpftool attempts
+		to autodetect the path for the base object, and passing
+		this option is optional. When the main BTF object is passed
+		through other handles, this option becomes necessary.
 
 EXAMPLES
 ========
 **# bpftool btf dump id 1226**
 
 ::
 
-  [1] PTR '(anon)' type_id=2
-  [2] STRUCT 'dummy_tracepoint_args' size=16 vlen=2
-          'pad' type_id=3 bits_offset=0
-          'sock' type_id=4 bits_offset=64
-  [3] INT 'long long unsigned int' size=8 bits_offset=0 nr_bits=64 encoding=(none)
-  [4] PTR '(anon)' type_id=5
-  [5] FWD 'sock' fwd_kind=union
+[1] PTR '(anon)' type_id=2
+[2] STRUCT 'dummy_tracepoint_args' size=16 vlen=2
+        'pad' type_id=3 bits_offset=0
+        'sock' type_id=4 bits_offset=64
+[3] INT 'long long unsigned int' size=8 bits_offset=0 nr_bits=64 encoding=(none)
+[4] PTR '(anon)' type_id=5
+[5] FWD 'sock' fwd_kind=union
```

Comment:
The examples won't need to be changed, we need some indent under the `::` marking the pre-formatting block, otherwise Docutils cannot know when the block ends.

Reply:
Noted. 
Skipping the examples.